tests/flow/enums/enum-export.js

```diff
@@ -0,0 +1,5 @@
+// @flow
```

Comment:
is this file from flow project? I think we only put tests from flow in this directory

Reply:
Yes, these are taken from the parser tests (selecting only the non-erroring files). They exercise all the possible cases of enums.

https://github.com/facebook/flow/tree/master/src/parser/test/flow/enums

Reply:
You're right thought that these come from `flow/src/parser/test/` rather than `flow/tests/`, and so synching in the future won't be correct, so I moved the tests into their own folder, `flow_enums`